infra/cifuzz/cifuzz.py

```diff
@@ -326,9 +328,14 @@ def get_files_covered_by_target(latest_cov_info, target_name, src_in_docker):
 
   affected_file_list = []
   for file in coverage_per_file:
-    if file['filename'].startswith(
-        src_in_docker) and file['summary']['regions']['count']:
-      affected_file_list.append(file['filename'].replace(src_in_docker, ''))
+    if not file['filename'].startswith(src_in_docker):
```

Comment:
Since we're doing path comparisons here, we should do os.path.normpath on both for robustness, in case there are redundant /../ etcs in either one.

Reply:
+1.